crowdstrike.sgnl.yaml

```diff
@@ -0,0 +1,582 @@
+displayName: "CrowdStrike"
```

Comment:
Given that we might have to have a few templates in the pretty near future, should this be 'CrowdStrike Identity Threat Protection'? (and the file-name?)